protected/tests/functional/AdminSiteAccessTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Functional test for only logged in admin user can access /site/admin.
+ */
+
+class AdminSiteAccessTest extends FunctionalTesting
+{
+    use BrowserSignInSteps;
+    use BrowserPageSteps;
+
+    public function testItShouldBeDisplayedToUsersWithAdminRole ()
```

Comment:
You should remove the space between the function name and brackets here and in the other 2 functions.